client/src/pages/lessons/index.tsx

```diff
@@ -163,13 +166,24 @@ const LessonItem = (props: {
   onDeleted: (id: string) => void;
 }) => {
   const { location, row, i, onDeleted } = props;
-  const [cookies] = useCookies(["user"]);
   const [anchorEl, setAnchorEl] = React.useState(null);
   const deleteMenuOpen = Boolean(anchorEl);
+  const context = useContext(ToggleContext);
+  const [cookies] = useCookies(["accessToken"]);
+
+  function canEdit() {
```

Comment:
make sure the logic of this function exists in only one place, e.g. maybe wherever the `Lesson` interface is defined

```typescript
export function canUserEditLesson(user: User, lesson: Lesson) : boolean
```

...and then this local, no-params `canEdit` just calls the above